packages/devextreme/js/__internal/scheduler/appointments/m_appointment_collection.ts

```diff
@@ -328,6 +328,10 @@ class SchedulerAppointments extends CollectionWidget {
     }
   }
 
+  _refreshActiveDescendant() {
+    // override to do nothing
```

Comment:
@roman-simionov do we really need this comment?

Reply:
Yes, otherwise it will be not obvious why we have empty method

Reply:
P.S., you mentioned wrong person